.github/workflows/docker-image.yml

```diff
@@ -2,7 +2,7 @@ name: Docker CI
 
 on:
   push:
-    branches: [ template_render, master ]
+    branches: [ multi-template_render_endpoint, master ]
```

Comment:
Is there a more generic way we can configure this?
Can we do `on: [push, pull_request]`?

Reply:
https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#onpushpull_requestbranchestags

^^ lots of options

I would suggest that future development uses branches off this repo, rather than using forks, and then we can configure the actions more precisely. Actions are not triggered by PRs from a fork, and I don't have the appropriate permissions to create branches in this repo ( 😿 ) or set up the Actions config.

Reply:
I added you as maintainer on this repo

Reply:
So can you change it to `on: [push, pull_request]`? My understanding is that will trigger the action on a PR from a fork or anywhere else.

Reply:
Done. I believe Jason is going to set up a shared KBASE_TEST_TOKEN secret for everything under `kbaseapps` so future PRs can use that.

Reply:
Tests pass on my branch: https://github.com/ialarmedalien/KBaseReport/runs/693820011?check_suite_focus=true